src/UI/IdentityPageModelConvention.cs

```diff
@@ -7,7 +7,8 @@
 
 namespace Microsoft.AspNetCore.Identity.UI
 {
-    internal class IdentityPageModelConvention<TUser> : IPageApplicationModelConvention where TUser : IdentityUser
+    internal class IdentityPageModelConvention<TUser> : IPageApplicationModelConvention
```

Comment:
The good news is this will be a lot simpler since it will probably only require that TUser is a class